tests/test_routes.py

```diff
@@ -108,4 +108,25 @@ def test_get_not_found(self):
         """It should not Read a Recommendation that is not found"""
         # Get a Recommendation that does not exist
         resp = self.client.get(f"{BASE_URL}/0")
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_delete(self):
+        """It should Delete a Recommendation if exists"""
+        # Create a new Recommendation
+        rec = make_recommendation(100, 200)
+        resp = self.client.post(
+            BASE_URL, json=rec.serialize(), content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        location = resp.headers.get("Location", None)
+        self.assertIsNotNone(location)
+
+        # Check that the location header was correct by deleting it
+        resp = self.client.delete(location, content_type="application/json")
```

Comment:
you need to added recommendation id to the URL, so that your handler knows which recommendation record to be deleted